docs/scenarios/kubernetes-edge.md

```diff
@@ -0,0 +1,143 @@
+# Connecting Edge Device to the Kubernetes Service
```

Comment:
I wonder if the file should be `microshift-edge.md` since it's specifically about microshift?

Reply:
It creates a mesh across Kubernetes clusters as well. Nodes from private Kubernetes cluster connected to Public Kubernetes Cluster nodes and can access service, so it kind of represent both hybrid and edge cloud scenarios.

Reply:
ok, fine to leave it as-is